app/commands/resumes/update.rb

```diff
@@ -0,0 +1,35 @@
+module Resumes
+  class Update
+    attr_accessor :user, :params, :current_resume_id
+
+    def initialize(user:, params:)
+      @user = user
+      @params = cast_to_attributes(params)
+      @current_resume_id = params[:current_resume_id]
+    end
+
+    def call!
+      ActiveRecord::Base.transaction do
+        # rubocop:disable Rails::SkipsModelValidations
```

Comment:
disabling this linter check that forbids `update_all` here because I've wrapped the database logic in a single transaction. Since subsequent database calls should fail noisily, there should always be a current resume that is assigned or the entire transaction should be rolled back.